src/main/java/core/basesyntax/utils/transaction/handlers/impl/BalanceOperation.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.utils.transaction.handlers.impl;
+
+import core.basesyntax.storage.impl.StorageImpl;
+import core.basesyntax.utils.transaction.FruitTransaction;
+import core.basesyntax.utils.transaction.handlers.OperationHandler;
+
+public class BalanceOperation implements OperationHandler {
+    public void perform(FruitTransaction transaction, StorageImpl storage) {
```

Comment:
According to the checklist, don't begin class or method implementation with an empty line. Please remove the redundant empty line at the beginning of the class definition.